tests/testthat/test-live.R

```diff
@@ -24,38 +24,38 @@ test_that("Scenario Test", {
   }
 
   res
-  
+
   getDoParWorkers()
-  
-  
+
+
   '%dopar%' <- foreach::'%dopar%'
   res <- foreach::foreach(i = 1:4) %dopar% {
     mean(1:i)
   }
-  
+
   res
-  
+
   results <- foreach(i = 1:1) %dopar% {
-    x <- try(library(IRanges),silent=TRUE)
-    x <- try(library(GenomeInfoDb),silent=TRUE)
-    x <- try(library(DESeq2),silent=TRUE)
+    x <- try(library(IRanges), silent = TRUE)
+    x <- try(library(GenomeInfoDb), silent = TRUE)
+    x <- try(library(DESeq2), silent = TRUE)
     #x <- sessionInfo()
     return(x)
   }
-  
+
   results
-  
-  
+
+
   #pragma message ("WARNING: use of OpenMP disabled; this compiler doesn't support OpenMP 3.0+")
-  
+
   # doAzureParallel::stopCluster(cluster)
```

Comment:
Remove all these comments?

Reply:
@zfengms, @brnleehng - I updated the container branch to fix a bunch of linter bugs. You can sync from there and most of the remaining items should go away.

Reply:
removed after merging latest from container branch